packages/editor/src/client/routes/contentList.tsx

```diff
@@ -0,0 +1,257 @@
+import React, {useEffect, useState} from 'react'
+import {Link, ButtonLink, ContentCreateRoute, ContentEditRoute, useRoute} from '../route'
+
+import {
+  useUnpublishContentMutation,
+  PageRefFragment,
+  useContentListQuery,
+  ContentListQuery,
+  ContentListRefFragment
+} from '../api'
+import {DescriptionList, DescriptionListItem} from '../atoms/descriptionList'
+import {useTranslation} from 'react-i18next'
+import {FlexboxGrid, Input, InputGroup, Icon, IconButton, Table, Modal, Button} from 'rsuite'
+import {getDeleteMutation} from '../utils/queryUtils'
+import {useMutation} from '@apollo/client'
+import {Content} from '@wepublish/api'
+import {EditorConfig} from '../interfaces/extensionConfig'
+const {Column, HeaderCell, Cell} = Table
+
+enum ConfirmAction {
+  Delete = 'delete',
+  Unpublish = 'unpublish'
+}
+
+const RecordsPerPage = 10
+
+export interface ArticleEditorProps {
+  readonly contentTypeList: EditorConfig
+}
+
+export function ContentList({contentTypeList}: ArticleEditorProps) {
+  const {current} = useRoute()
+  const type = (current?.params as any).type || ''
+  const [filter, setFilter] = useState('')
+
+  const [isConfirmationDialogOpen, setConfirmationDialogOpen] = useState(false)
+  const [currentContent, setCurrentContent] = useState<Content>()
+  const [confirmAction, setConfirmAction] = useState<ConfirmAction>()
+
+  const config = contentTypeList.contentModelExtension.find(config => {
+    return config.identifier === type
+  })
+  if (!config) {
+    throw Error(`Content type ${type} not supported`)
+  }
+
+  const [articles, setArticles] = useState<any[]>([])
+
+  const [unpublishArticle, {loading: isUnpublishing}] = useUnpublishContentMutation()
+  const [deleteContent, {loading: isDeleting}] = useMutation(getDeleteMutation(config))
+
+  const listVariables = {type, filter: filter || undefined, first: RecordsPerPage}
+  const {data, fetchMore, loading: isLoading, refetch} = useContentListQuery({
+    variables: listVariables,
+    skip: !type,
+    fetchPolicy: 'network-only'
+  })
+
+  const {t} = useTranslation()
+
+  useEffect(() => {
+    if (data?.content._all.list.nodes) {
+      setArticles(data?.content._all.list.nodes.map(a => a.content))
+    }
+  }, [data?.content._all.list])
+
+  function loadMore() {
```

Comment:
i strongly recommend using the the pagination components as in the article and page list. I already commented about this in the API. As soon as you want to sort the `fetchMore` is useless. And